### DISCUSSION TITLE
converting PCRE to tintin

### DISCUSSION THREAD

**Discussion by nacjacks** [scandum/tintin#129] [Script help]
I have a working regular expression https://regex101.com/r/ll9vrk/1 matching prompts from a mud. When I convert it to a tintin++ action it does not fire. I believe it is in `{\[Pos:{standing|sitting|resting|sleeping}\]{\s\[.+\]}?}` but not sure.

`#ACTION {^{[\d]*H}\s{[\d]*V}\s{[\d]*X}\s{.+\%}\s{[\d]*C}\s{\[Pos:{standing|sitting|resting|sleeping}\]{\s\[.+\]}?}\s{Mem:\d+\s}?{T:\d*}\s{.*}}
{
    #SHOW {action triggered};
}`

**Comment by scandum** [comment]
You'd have to either use:
```
#action {%!{^([\d]*H)\s([\d]*V)\s([\d]*X)\s(.+\%)\s([\d]*C)\s(\[Pos:(standing|sitting|resting|sleeping)\](\s\[.+\])?)\s(Mem:\d+\s)?(T:\d*)\s(.*)}}
```
or properly convert it to tintin's regex format.
